android/src/main/java/org/reactnative/camera/events/BarcodesDetectedEvent.java

```diff
@@ -72,6 +72,7 @@ private WritableMap serializeEventData() {
       Barcode barcode = mBarcodes.valueAt(i);
       WritableMap serializedBarcode = Arguments.createMap();
       serializedBarcode.putString("data", barcode.displayValue);
+      serializedBarcode.putString("rawValue", barcode.rawValue);
```

Comment:
This is already implemented on iOS. 
Check out: https://github.com/react-native-community/react-native-camera/blob/22533ed8e8c47f17ed7ca28f5879064fc2572a7f/ios/RN/RNCamera.m#L884

Please use the same name `rawData` instead of `rawValue`. What happens if the barcode does not contain a raw value? Will putString result in `rawData` being `null` or will the application crash?

Reply:
Also implemented on android already (I think?) - https://github.com/Clever/react-native-camera/blob/a3afef1ab25da9337f3d9cf0ef9259063348b6e5/android/src/main/java/org/reactnative/camera/events/BarCodeReadEvent.java#L77

It's not immediately clear to me what the difference between thee 2 files are, but we should make sure value returned is the same in each